files/class-vip-filesystem.php

```diff
@@ -490,7 +490,12 @@ public function update_attachment_meta() {
 			return;
 		}
 
-		$attachments = $updater->get_attachments( $start_index, $end_index );
+		do {
+			$attachments = $updater->get_attachments( $start_index, $end_index );
```

Comment:
I think we'll need to move the `$start_index > $max_id` check in here otherwise we risk getting into an infinite loop.

Reply:
That's probably a good idea